app-games/src/main/java/com/aptoide/android/aptoidegames/updates/presentation/UpdatesScreen.kt

```diff
@@ -76,35 +86,94 @@ fun UpdatesScreen(
 
 @Composable
 fun NoUpdatesScreen() {
+  val (autoUpdateGames, toggleAutoUpdate) = rememberAutoUpdate()
+  val showAutoUpdateToggle = remember { mutableStateOf(false) }
+  LaunchedEffect(key1 = autoUpdateGames) {
+    if (!autoUpdateGames) showAutoUpdateToggle.value = true
+  }
+
   Column(
     modifier = Modifier.fillMaxSize(),
     verticalArrangement = Arrangement.Center,
     horizontalAlignment = Alignment.CenterHorizontally
   ) {
-    Image(
-      modifier = Modifier
-        .padding(bottom = 88.dp, start = 16.dp, end = 16.dp)
-        .fillMaxWidth(),
-      imageVector = getNoUpdates(Palette.Primary, Palette.White, Palette.GreyLight),
-      contentDescription = null,
-    )
-    Text(
-      modifier = Modifier.padding(horizontal = 40.dp),
-      text = stringResource(R.string.update_up_to_date_title),
-      style = AGTypography.Title,
-      color = Palette.White,
-      maxLines = 2,
-      overflow = TextOverflow.Ellipsis,
-      textAlign = TextAlign.Center,
+    if (showAutoUpdateToggle.value) {
```

Comment:
Would it be easier to apply a spacer between the items instead of passing a padding as a parameter?